rest/rest-resources/src/main/java/org/apache/brooklyn/rest/resources/LocationResource.java

```diff
@@ -129,6 +139,11 @@ public LocationSummary get(String locationId, String fullConfig) {
     }
 
     public LocationSummary get(String locationId, boolean fullConfig) {
+        if (!Entitlements.isEntitled(mgmt().getEntitlementManager(), Entitlements.SEE_LOCATION, Entitlements.StringAndArgument.of(locationId+"| fullConfig: "+fullConfig, "see"))) {
+            throw WebResourceUtils.forbidden("User '%s' is not authorized to see locations",
```

Comment:
Message should say `User '%s' is not authorized to see the location '%s'`
Please also check the parameters for the templated message